src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerManager.AnalyzerExecutionContext.cs

```diff
@@ -339,7 +339,7 @@ private void MarkSymbolEndAnalysisPending_NoLock(ISymbol symbol, ImmutableArray<
                 _lazyPendingSymbolEndActionsOpt[symbol] = (symbolEndActions, symbolDeclaredEvent);
             }
 
-            [Conditional("DEBUG")]
+            [Conditional("DEBUG_ISSUE_30309")]
```

Comment:
>_ISSUE_30309 [](start = 31, length = 12)

This should be reverted as I have a PR to fix the assert: [https://github.com/dotnet/roslyn/pull/30368](https://github.com/dotnet/roslyn/pull/30368)

Reply:
Fixed in 5d1008485e792881a0860fdffa3d45e65efcea98